src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
Just to clarify, these options are under the github key inside of the forge config. They're not at the top level. I'm not sure what else is in store for publish targets (mas?) but these are, at least at this point, specific to fields in GitHub's release API.